backend/src/models/__tests__/game.model.test.ts

```diff
@@ -2,6 +2,10 @@ import mongoose from "mongoose";
 import { GameModel, SetupType } from "../../models";
 
 describe("Game Model", () => {
+  let gameCode = 42069;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let gameData: any;
+
```

Comment:
It looks like you did change the model slightly so these test changes could be okay. I feel like the service tests should be part of a separate PR (see my other comment on the service test)

Reply:
Yeah I agree - it would've been nice to have split the services & handlers up although unfortunately short on time 😥